x-pack/plugin/ent-search/src/test/java/org/elasticsearch/xpack/entsearch/engine/EngineIndexServiceTests.java

```diff
@@ -88,7 +88,7 @@ private void checkAliases(Engine engine) {
     public void testUpdateEngine() throws Exception {
         {
             final Engine engine = new Engine("my_engine", new String[] { "index_1", "index_2" }, null);
-            IndexResponse resp = awaitPutEngine(engine);
+            IndexResponse resp = awaitPutEngine(engine, false);
             assertThat(resp.status(), equalTo(RestStatus.CREATED));
             assertThat(resp.getIndex(), equalTo(ENGINE_CONCRETE_INDEX_NAME));
 
```

Comment:
Can you add a test that fails if the document already exists and create is true? Would be nice to have a yaml test too.